CppSamples/DisplayInformation/ShowGrid/README.metadata.json

```diff
@@ -25,13 +25,12 @@
         "MGRSGrid",
         "MapView",
         "SimpleLineSymbol",
+        "SceneView",
         "TextSymbol",
         "USNGGrid",
         "UTMGrid"
     ],
-    "redirect_from": [
-        "/qt/latest/cpp/sample-code/sample-qt-displaygrid.htm"
-    ],
```

Comment:
❓ do we want to remove this? presumably not. people or websites might have bookmarks or links to the old sample, and they should get redirected to the new one